standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/model/MMetastoreDBProperties.java

```diff
@@ -21,6 +21,8 @@ public class MMetastoreDBProperties {
   private String propertyKey;
   private String propertyValue;
   private String description;
+  private byte[] propertyContent;
```

Comment:
Should we create a column in underlying database that maps to this field?

Reply:
We actually do add a column which is a blob; the update scripts are lacking.

Reply:
Adding the schema updating scripts.